client/src/components/Login.tsx

```diff
@@ -1,7 +1,15 @@
 import React, { useState } from "react";
 
-import { Grid, Paper, Avatar, TextField, Button, Box } from "@material-ui/core";
-import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+
+import Grid from "@mui/material/Grid";
```

Comment:
Not sure why you split the imports up like this, I tested it and they should still work fine if you do:

`import { ... } from "@mui/material

Reply:
The build time becomes insanely slow when I do that

Reply:
I looked it up and I think it has something to do with "tree-shaking".

Reply:
> What we've learned is that in order to take advantage of tree shaking, you must...
> 
> Use ES2015 module syntax (i.e. import and export).
> Ensure no compilers transform your ES2015 module syntax into CommonJS modules (this is the default behavior of the popular Babel preset @babel/preset-env - see the documentation for more details).
> Add a "sideEffects" property to your project's package.json file.
> Use the production mode configuration option to enable various optimizations including minification and tree shaking.

We could do this but it's probably just easier to separate imports.

Reply:
"Tree shaking" removes unused imports and makes your bundle size smaller (the JS file that's built) which makes for quicker asset download times and a better user experience overall.